packages/react-i18n/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+- Add documentation about React Native support ([#1415](https://github.com/Shopify/quilt/pull/1415))
```

Comment:
You actually don't need this as we don't publish a new version for documentation or testing changes.